data/coalition/lunarium intro.txt

```diff
@@ -2012,11 +2012,11 @@ mission "Lunarium: Join"
-			`	He leaves without saying anything else, leaving everyone else confused with his abrupt departure. "Much to take in at once, it must be for him, in particular. A Heliarch he once was, after all," Tummug says. "Get to work on perfecting the footage, and readying it for release to the public, we should."`
+			`When you and Oobat reach the Lunarium's complex, you're welcomed back by Chiree, Elirom and Tummug. "Tell you I did that come back safely, they would," Tummug says to Elirom. "So, good enough to use in our efforts, was it?"`
+			`	Oobat shows them the recording, satisfaction clear on their faces as they listen to the Quarg. The mood darkens when the last question appears, however. Once it ends, Elirom stands up and heads to the nearest wall, slamming an arm onto it. Chiree sighs, and turns to Tummug, "Worth using still, is the footage?"`
+			`	"Up until that one? Yes, of course. Need to edit it for a cleaner, quicker video, we do, but a great piece it will be," he says. "That last part though, go as we expected, it did not."`
+			`	"Only prove the Heliarchs right, that last question would," Elirom chimes in. "Matter not, it does, if a thousand times the footage disproves their narrative. If even one thing, pick and spin as proof of their side they can, fruitless, the effort will be." He pauses, then hits the wall again, with less force. "Play it again, could you?" Oobat obliges, and plays the recording once again. Elirom has it paused a few times over the course of the video, breaking out a device to write down some notes. Once he's finished, he has the last portion repeated a few times still, seeming particularly interested in the Quarg's changing pigment. "Onyx black..." he ponders out loud, "what was, again, the emotion of that color?" Snapping out of the trance only after a few more repetitions, he hands Oobat the notes. "Right they were about the Quarg wanting to take the rings back... A shame it is, that use that point against them we cannot. Though, maybe, if interested in the rings they still are after so long..."`
```

Comment:
Was removing the "it" from the "the emotion of that color, what was it again?" bit intentional? The new version reads a bit weird without it.

Reply:
I see. The original change is fine then I think. Elirom's more thinking aloud to himself, rather than explaining out loud what he's trying to remember, so I think the original change fits better.